pages/common/lstopo.md

```diff
@@ -0,0 +1,24 @@
+# lshw
```

Comment:
```suggestion
# lstopo
```

Reply:
Ah oops. I started off by copying `lshw.md` so that's how that happened 😅.